sw/applications/example_set_retentive_external_ram_blocks/main.c

```diff
@@ -1,5 +1,5 @@
 // Copyright EPFL contributors.
-// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.\n
```

Comment:
shouldn't we change this application to return EXIT_SUCCESS in case NUM_BANKS = 2? 

AFAIK, the CI uses MEMBANKS = 3, can we change it to 2?

Reply:
The CI is currently using 6 (at the time freertos required 5). It can be reduced to 2. We could make an issue, but we might want to review this before going for the simulation CI. 

Regarding this app, this works with 2mb (my comment at the top of this PR was wrong). 

Also, good catch with that `\n`

Reply:
I tried it with 2mb and it worked properly and returned `SUCCESS`

Reply:
but here it is written return failire https://github.com/JuanSapriza/x-heep/blob/bf52f256048d5ad0e904f58debef9d86673e9407/sw/applications/example_set_retentive_ram_blocks/main.c#L70

Reply:
but this is the example set retentive EXTERNAL ram blocks, which i guess makes sense it does not have this issue. 

What i can do is change the name to `example_set_retentive_ram_blocks_external` so the difference is left at the end of the name to avoid further confusions

Reply:
Done